packages/client/base/package.json

```diff
@@ -20,6 +20,7 @@
     ],
     "scripts": {
         "build": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --minify --dts --sourcemap",
+        "build:prod": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --minify --dts",
```

Comment:
its really ugly to have to repeat the commands like this, but its bc we arent currently able to share configs between projects, which we will eventually do, then the duplication can be removed